zest/releaser/towncrier_hooks.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+from zest.releaser import utils
+import logging
+import os
+import pkg_resources
+import toml
+
+# towncrier might become a conditional dependency of zest.releaser,
+# for example in the recommended list.
+# If we split this into a separate package, it should be a hard dependency.
+try:
+    pkg_resources.get_distribution('towncrier')
+except pkg_resources.DistributionNotFound:
+    HAS_TOWNCRIER = False
+else:
+    HAS_TOWNCRIER = True
+
+
+logger = logging.getLogger('towncrier_hooks')
+TOWNCRIER_MARKER = '_towncrier_applicable'
+TOWNCRIER_CONFIG_FILE = 'pyproject.toml'
+
+
+def load_config():
+    fn = os.path.join(TOWNCRIER_CONFIG_FILE)
```

Comment:
What does "fn" mean? It sounds like "function". Better have a more descriptive name.

Reply:
This part was copied from the `towncrier` code, and only slightly adapted. I can indeed improve it.